site/src/main/.vuepress/config.js

```diff
@@ -438,8 +438,8 @@ var config = {
 						title: '5-DataQuery',
 						children: [
 							['5-DataQuery/1-DataQuery','DataQuery'],
-							['5-DataQuery/2-SeriesReader','SeriesReader'],
-							['5-DataQuery/3-ModificationHandle','ModificationHandle'],
+							['5-DataQuery/2-QueryFundamentals','QueryFundamentals']
+							['5-DataQuery/3-SeriesReader','SeriesReader'],
 							['5-DataQuery/4-RawDataQuery','RawDataQuery'],
```

Comment:
I think you miss a comma after`'QueryFundamentals']`.

Reply:
Fixed, thanks